tests/phpunit/integration/api/term-meta/beans-term-meta/delete.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Tests for _beans_term_meta::delete()
+ *
+ * @package Beans\Framework\Tests\Integration\API\Term_Meta
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\Term_Meta;
+
+use Beans\Framework\Tests\Integration\API\Term_Meta\Includes\Beans_Term_Meta_Test_Case;
+use _Beans_Term_Meta;
+
+require_once BEANS_THEME_DIR . '/lib/api/term-meta/class-beans-term-meta.php';
+require_once BEANS_THEME_DIR . '/lib/api/term-meta/functions-admin.php';
+require_once dirname( __DIR__ ) . '/includes/class-beans-term-meta-test-case.php';
+
+/**
+ * Class Tests_BeanTermMeta_Delete
+ *
+ * @package Beans\Framework\Tests\Integration\API\Term_Meta
+ * @group   api
+ * @group   api-term-meta
+ */
+class Tests_BeansTermMeta_Delete extends Beans_Term_Meta_Test_Case {
+
+	/**
+	 * Tests _beans_term_meta::delete() should remove term meta option from db options table when called.
```

Comment:
1. Fix the class name `_Beans_Term_Meta::delete()`.
2. Remove the `when called` as it's not necessary.